src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesApiServiceTest.kt

```diff
@@ -75,9 +75,9 @@ internal class ActivitiesApiServiceTest {
             },
             "startDate": "2022-12-30",
             "endDate": "2022-12-30",
-            "allocatedTime": "2022-12-30T14:03:06.365Z",
+            "allocatedTime": "2022-12-30T14:03:06",
```

Comment:
Activities are sending in this format which dolesn't parse to OffsetDateTime - so we're now using LocalDateTime instead (which is the same they  use on their side)